internal/namespaces/rdb/v1/testdata/test-update-user-simple.golden

```diff
@@ -0,0 +1,11 @@
+🎲🎲🎲 EXIT CODE: 0 🎲🎲🎲
+🟩🟩🟩 STDOUT️ 🟩🟩🟩️
+Name      cli-test
+IsAdmin   true
+Password  Newp1ssw0rd!
+🟩🟩🟩 JSON STDOUT 🟩🟩🟩
+{
+  "name": "cli-test",
+  "is_admin": true,
+  "Password": "Newp1ssw0rd!"
```

Comment:
use password to stay consistent with the case of the other component of the JSON